packs/feats/death-roll.json

```
@@ -31,7 +31,18 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Death Roll",
+                "predicate": [
+                    "target:condition:grabbed"
+                ],
+                "selector": "jaws-attack-roll",
+                "type": "circumstance",
+                "value": 2
+            }
```

```suggestion
            {
                "key": "FlatModifier",
                "predicate": [
                    "target:condition:grabbed"
                ],
                "selector": "jaws-attack-roll",
                "type": "circumstance",
                "value": 2
            }
```
This isn't a sufficient predicate, the bonus should only be active against creatures you have grabbed so it just needs to be a toggle. Also we don't need to provide labels that match the item name, and if we need labels that don't match we need to localize them.